compiler/src/lnntoamm/Expr.ts

```diff
@@ -92,28 +93,202 @@ export default abstract class Expr {
   }
 
   static fromAssignablesAst(ast: LPNode, metadata: MetaData): [Stmt[], Expr] {
+    const asts = ast.getAll();
     // break it up so that we're only working on one base assignable list or operator at a time.
-    let operated = ast.getAll().map(work => {
+    let operated: Array<[Stmt[], Expr] | Operator[]> = asts.map(work => {
       work = work.get('withoperators');
       if (work.has('baseassignablelist')) {
         return Expr.fromBaseassignablelist(work.get('baseassignablelist'), metadata);
       } else if (work.has('operators')) {
-        return TODO('operators');
+        // TODO: this won't work with operators associated with interfaces.
+        // Will have to iterate through all of the interfaces in-scope and collect
+        // the applicable types as well
+        const op = work.get('operators').t.trim();
+        let operators = metadata.scope.get(op) as Operator[];
+        if (operators === null) {
+          console.log(metadata.scope);
+          throw new Error(`can't find operator ${op}`);
+        } else if (!isOpArray(operators)) {
+          // sanity check
+          throw new Error(`somehow ${op} isn't an operator?`);
```

Comment:
It may make sense to dump `operators` here, as well if this ever actually hits, because knowing what weird thing got in here would help a lot when tracing down the error.